eng/deployment/product-construction-service-deploy.ps1

```diff
@@ -92,25 +92,28 @@ else
 Write-Host "Stopping the service from processing new jobs"
 StopAndWait -pcsStatusUrl $pcsStatusUrl -pcsStopUrl $pcsStopUrl -authenticationHeader $authenticationHeader
 
-# deploy the new image
+# Kick off the deployment of the new image
```

Comment:
@dkurepa what do you think about this? Will this speed up things?

Reply:
Example: https://dev.azure.com/dnceng/internal/_build/results?buildId=2517195&view=logs&j=4032b39f-a33d-55dd-373e-29d781cedb05&t=2dc7e74f-ec8e-548e-5f36-a22abb127a58

Reply:
I think it would a bit, because the service will be able to initialize while we're deploying jobs.
You'll just want to move this below line 101, that's where we initialize the `$newImage` var

Reply:
Ah, good catch. Fixed! And also improved the sleeping